### PR TITLE
full support for file attachments

### PR DESCRIPTION
**Pull request by nvdk**
This is a draft PR only to be merged when all work around file attachments is done. The futurenow branch can be tested on https://futurenow.gelinkt-notuleren.lblod.info/

### CODE REVIEW

**Comment by nvdk** [comment]
TODO at the very least:
- linting issues
- resolve conflicts

**Comment by lagartoverde** [comment]
Solved the conflicts, and linting problems got magically solved so I think it can be merged

File: package.json [hunk 1]
```diff
@@ -115,6 +115,9 @@
     "tracked-built-ins": "^1.0.2",
     "tracked-toolbox": "^1.2.3"
   },
+  "dependencies": {
+    "@lblod/ember-vo-mu-file-upload": "git://github.com/lblod/ember-vo-mu-file-upload.git#feature/configurable-model-name"
```
Comment (nvdk) [inline package.json:119]: do we need this branch? looks like we don't but perhaps I'm missing something

Comment (lagartoverde) [reply package.json:119]: @Asergey91

**Pull request merged**